Jenkinsfile

```diff
@@ -89,14 +100,23 @@ pipeline {
                 assert condition : "Build fails because it is not a PR/Tag/Branch"
             }
 
+            CI_STATE.SELF.COMPLIANCE.ARGS += " -S $CI_STATE.SELF.REPORT_SHA "
+            CI_STATE.SELF.COMPLIANCE.ARGS += " --commits $COMMIT_RANGE "
+            // CI_STATE.SELF.COMPLIANCE.ARGS += " --exclude-module Kconfig "
```

Comment:
Is this dead code or something that should remain for the future?

Reply:
it is a test-suite that had to be disabled before, but now it passes.    I removed it.